src/components/portfolio/DetailsVolatilityGraph.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import {
+  makeStyles,
+  createStyles,
+  Theme,
+  useTheme,
+} from '@material-ui/core/styles';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import HotelIcon from '@material-ui/icons/Hotel';
+import Grid from '@material-ui/core/Grid';
+import VolatilityLineEntry from './VolatilityLineEntry';
+import { Position } from '../../portfolio/APIClient';
+
+type DetailsVolatilityGraphProps = {
+  positions: Position[];
+};
+
+const DetailsVolatilityGraph: React.FC<DetailsVolatilityGraphProps> = ({
+  positions,
+}) => {
+  const useStyles = makeStyles(({ palette }: Theme) =>
+    createStyles({
+      text: {
+        color: palette.primary.contrastText,
+      },
+      line: {
+        width: '90%',
+        height: '3px',
+        margin: 'auto',
+        'background-color': palette.primary.contrastText,
+        position: 'relative',
+        display: 'flex',
+      },
+      lineMiddle: {
+        height: '3rem',
+        width: '0.25rem',
+        'background-color': palette.primary.contrastText,
+        margin: 'auto',
+      },
+      marketAvg: {
+        position: 'absolute',
+        top: '50%',
+        left: '50%',
+        transform: 'translate(-50%, -100%)',
+        display: 'flex',
+        flexDirection: 'column',
+      },
+      lineMiddleText: {
+        color: palette.primary.contrastText,
+        marginBottom: '0.5rem',
+      },
+      wrapper: {
+        height: '250px',
+      },
+    })
+  );
+
+  type VolatilityLineData = {
+    [symbol: string]: number;
+  };
+
+  const theme = useTheme();
+  const classes = useStyles();
+
+  const sortedStocks: VolatilityLineData = {};
+
+  if (positions) {
+    Object.values(positions)
+      .map((p) => p.stock)
+      .sort((a, b) => (a.volatility > b.volatility ? -1 : 1))
```

Comment:
I thought we wanted to display the volatility of the stocks wit the highest monetary share in the portfolio, not the one with the highest volatility?

Reply:
oh my bad, i misunderstood then - let me fix that real quick